utils/systemd-redis_server.service

```diff
@@ -0,0 +1,41 @@
+# example systemd service unit file for redis-server
```

Comment:
About the license tag, see here https://softwareengineering.stackexchange.com/a/381907/49834

Reply:
Sorry for my late response - my father passed away last week, and I couldn't get myself to work on stuff like this since then. I'll consider your suggestions and get back to you and this issue/PR soon.